.npmrc

```diff
@@ -0,0 +1 @@
+shell-emulator=true
```

Comment:
Why do we need this now?

Is it to "support cross platform shell commands", in which case did you find it useful? I'm curious 👀

Reply:
Yes, I found it "nice" for windows users but totally unrelated, I can remove it if needed.